src/manifold/src/quickhull.cpp

```diff
@@ -32,33 +32,11 @@ double defaultEps() { return 0.0000001; }
 ConvexHull QuickHull::getConvexHull(const std::vector<glm::dvec3>& pointCloud,
                                     bool CCW, bool useOriginalIndices,
                                     double epsilon) {
-  VertexDataSource vertexDataSource(pointCloud);
+  manifold::Vec<glm::dvec3> vertexDataSource(pointCloud);
```

Comment:
Also, will this change the timing? You are copying the points into a `Vec`. Should we modify the input `pointCloud` to `VecView`?

Reply:
> @elalish I vaguely recall you said we should move the quickhull thing into the `manifold` namespace right?
> 
> If we move the namespace, we don't need this `manifold::` namespace specifier.

That was regarding removing the namespace for quickhull right? So, should I also wrap the code in namespace manifold?

Reply:
> Also, will this change the timing? You are copying the points into a `Vec`. Should we modify the input `pointCloud` to `VecView`?

I tried modifying the input 'pointCloud' to VecView, but it was showing the error that VecView had a protected constructor. I can try to surpass that though, I'll give it one more try.

Reply:
Also, `optimizedVertexBuffer` is a `unique_ptr` so, it won't be compatible with `VecView` right?, so when I'm calling

`vertices = manifold::VecView<glm::dvec3>(*optimizedVertexBuffer);` it won't work right?

Reply:
1. Just change the namespace for quickhull to manifold.
2. Can you show me the modification and error message about protected constructor?
3. `optimizedVertexBuffer` is an internal buffer, you want `Vec` for that.

Reply:
In terms of modification all I'm doing is since we want to make pointCloud VecView, I make some Vec's VecView's for the code to compile (without incorrect function definitions) but this error persists

Reply:
Alright, I'll commit the changes I've made to the Mesh structure first and then I'll work on this, to separate the commits